lib/oci_utils/impl/oci-iscsi-config-main.py

```diff
@@ -164,49 +207,56 @@ def ocid_refresh(wait=False, debug=False):
     ----------
     wait: bool
        Flag, wait until completion if set.
-    debug: bool
-       Flag, write the result to standard error if set.
 
     Returns
     -------
         bool
             True on success, False otherwise.
     """
-    if debug:
-        debug_opt = ['--debug']
-        wait = True
-    else:
-        debug_opt = []
-
-    # TODO : what happen with (wait=True and debug=False) ?
-
+    _cmd = ['/usr/libexec/ocid','--refresh', 'iscsi']
+    if wait:
+        _cmd.append('--no-daemon')
     try:
-        if wait:
-            output = subprocess.check_output(['/usr/libexec/ocid',
-                                              '--no-daemon',
-                                              '--refresh',
-                                              'iscsi'] + debug_opt,
-                                             stderr=subprocess.STDOUT)
-        else:
-            output = subprocess.check_output(['/usr/libexec/ocid',
-                                              '--refresh',
-                                              'iscsi'] + debug_opt,
-                                             stderr=subprocess.STDOUT)
-        _logger.debug(str(output))
+        output = subprocess.check_output(_cmd, stderr=subprocess.STDOUT)
+        if _logger.isEnabledFor(logging.DEBUG):
+            _logger.debug('ocid run output: %s' % str(output))
         return True
-    except subprocess.CalledProcessError:
+    except subprocess.CalledProcessError as e :
+        _logger.debug('launch of ocid failed : %s',str(e))
```

Comment:
shouldn't this be a _logger.error?

Reply:
why not. I've followed logging already present . I'll change it